Modelica/Resources/C-Sources/ModelicaStandardTables.c

```diff
@@ -6222,8 +6246,17 @@ static size_t findColIndex(_In_ const double* table, size_t nCol, size_t last,
     return i0;
 }
 
+static size_t findColIndex(_In_ const double* table, size_t nCol, size_t last, double x) {
+    return findColIndex2(table, nCol, last, x, 0.0);
+}
+
+
 /* ----- Internal check functions ----- */
 
+static int isLess(double x, double dx, double val) {
+    return x < val || (x == val && dx < 0);
```

Comment:
Checking floating point numbers for equality is not the safest thing. We tried to avoid it at all places in MSL C code. Can we use `isNearlyEqual` or do better otherwise?

Reply:
But here the idea is to check less than, right? If it's slightly smaller it would have been true already. And if it's slightly bigger it shouldn't be less than.

Reply:
Hmh, maybe rename `isLess` again to something like `isLessOrEqualIfNegativeSlope`?

Reply:
Something like that has been done.